client/src/services/course.ts

```diff
@@ -780,4 +780,6 @@ export interface CrossCheckPairs {
   comment: string;
   score: number;
   id: number;
+  submittedDate: string;
```

Comment:
please remove this type and just use `CrossCheckPairDto` from `api`. It seems we don't need this type anymore

Reply:
done